collect_app/src/main/java/org/odk/collect/android/utilities/FileUtils.java

```diff
@@ -587,20 +584,6 @@ public static String getFileExtension(String fileName) {
     public static void grantFilePermissions(Intent intent, Uri uri, Context context) {
         intent.addFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
         intent.addFlags(Intent.FLAG_GRANT_WRITE_URI_PERMISSION);
-
```

Comment:
Please get rid of the comment below since the explanation won't be needed now.

Reply:
Yes Correct!